README.md

```diff
@@ -1,87 +1,5 @@
-
-## Installation
-
-### CocoaPods
-
-To install SuperDelegate in your iOS project running Xcode 7 with [CocoaPods](http://cocoapods.org), add the following to your `Podfile`:
-
-```
-platform :ios, '8.0'
-pod 'SuperDelegate', '~> 0.9.0'
-```
-
-Xcode 7 and Swift 2 projects should use 0.7.0
-
-### Carthage
-
-To install SuperDelegate in your iOS project with [Carthage](https://github.com/Carthage/Carthage), add the following to your `Cartfile`:
-
-```ogdl
-github "Square/SuperDelegate"
-```
-
-Run `carthage` to build the framework and drag the built `SuperDelegate.framework` into your Xcode project.
-
-### Swift Package Manager
-
-To install SuperDelegate in your iOS project with [Swift Package Manager](https://github.com/apple/swift-package-manager), the following definition can be added to the dependencies of your `Project`:
-
-```swift
-  .Package(
-    url: "https://github.com/square/SuperDelegate.git",
-    versions: Version(0,9,0)..<Version(1,0,0)
-  ),
-```
-
-### Submodules
-
-To use git submodules, checkout the submodule with `git submodule add git@github.com:Square/SuperDelegate.git`, drag SuperDelegate.xcodeproj to your project, and add SuperDelegate as a build dependency.
-
-## Getting Started
-
-```swift
-public class AppDelegate: SuperDelegate, ApplicationLaunched
-```
-
-Have your `UIApplicationDelegate` subclass `SuperDelegate` and conform to the `ApplicationLaunched` protocol. SuperDelegate guarantees to call `setupApplication()` only once per application launch, before making any other delegate calls. SuperDelegate also guarantees that `loadInterfaceWithLaunchItem(_:)` will be called once, when your app is brought to the foreground for the first time after it is launched.
-
-
-## Adopting more AppDelegate features
-
-To opt into more AppDelegate features, have your `AppDelegate` class conform to the associated protocol. For example, making your `AppDelegate` class conform to `LocalNotificationCapable` will give your app the ability to post local notifications.
-
-
-## Requirements
-
-* Xcode 7.0 or later.
-* iOS 8 or later.
-
-## Versions
-* 0.8.* – Swift 2.3 and Xcode 7.3+.
-* 0.9.* – Swift 3.0 and Xcode 8+.
-* 1.0 – Swift 3.0, Xcode 8+, and iOS 10 SDK adoption. Currently in development.
-
-## Contributing
-
-We’re glad you’re interested in SuperDelegate, and we’d love to see where you take it. Please read our [contributing guidelines](Contributing.md) prior to submitting a Pull Request.
-
-Thanks, and happy delegating!
+This library has been deprecated and is no longer under active development.
+The original README can be found in [README_OLD](README_OLD.md), but we
+would recommend moving away from using this library in the future.
```

Comment:
Should we include a section asking for new maintainers? If someone wants to maintain this, I'd be in favor of transferring ownership.